dvc/repo/experiments/base.py

```diff
@@ -95,6 +96,15 @@ def __init__(self, name):
         self.name = name
 
 
+class GitRemoteAuthError(GitAuthError):
```

Comment:
This seems like it might not belong in `experiments`?

Maybe `scm.base.GitAuthError` should just have the more detailed error message by default, and then rather than having the per-command error link and a repeated "git auth requirements" docs section in every `exp` command that does remote git operations, we could just link to a single troubleshooting entry (https://dvc.org/doc/user-guide/troubleshooting) page regarding git auth instead?

Reply:
Makes sense.